index.js

```diff
@@ -0,0 +1,5 @@
+function add() {
+  const x = 11;
+  return x;
+}
+add();
```

Comment:
- [ ] Please delete this file. It is of no relevance to this project. :100: Kindly make sure to always follow the project requirements :+1: